src/applications/personalization/profile-2/msw-mocks.js

```diff
@@ -464,20 +450,14 @@ export const editEmailAddressSuccess = () => {
   ];
 };
 
+// Sets up the responses needed to mack a successful email deletion. In
+// particular it mocks the `GET user/` response so that the email address no
+// longer exists
 export const deleteEmailAddressSuccess = [
-  rest.delete(`${prefix}/v0/profile/email_addresses`, (req, res, ctx) => {
+  rest.delete(`${prefix}/v0/profile/*`, (req, res, ctx) => {
```

Comment:
Do we not need `email_addresses` here?

Reply:
Or if we want to make this a generic method, at least rename it?

Reply:
Yeah I should have totally renamed this. Good catch!

Reply:
Actually this one should keep this name and be kept specific to the `email_addresses` route.